### PR TITLE
Fix typos in Jobs documentation

### PR DESCRIPTION
**Pull request by RouganStriker**
Refs #SDE-????

TODO:
- [x] Set milestone label in pull request (SDE version)
- [x] Updated changelog.md
- [x] Completed documentation for the new endpoint.
- [x] Feature has been merged into SDE develop.

### CODE REVIEW

**Comment by RouganStriker** [comment]
This is just to fix 3 typos in the docs, does it really need a checklist @sibartlett =P?

**Comment by sibartlett** [comment]
@kchan use your discretion :)

File: source/includes/jobs.md [hunk 1]
```diff
@@ -197,7 +197,7 @@ Content-Type: application/json
 
 Will submit the results of an ALM job.
 
-**`POST /api/v2/jobs/alm/`**
+**`POST /api/v2/projects/1/jobs/alm/`**
```
Comment (harry-kim) [inline source/includes/jobs.md]: no /projects

File: source/includes/jobs.md [hunk 2]
```diff
@@ -465,7 +465,7 @@ Content-Type: application/json
 
 Will submit the results of an analysis job.
 
-**`POST /api/v2/jobs/analysis/`**
+**`POST /api/v2/projects/1/jobs/analysis/`**
```
Comment (harry-kim) [inline source/includes/jobs.md]: no /projects/

**Pull request merged**